package/cluster/composition.yaml

```
@@ -7,13 +7,24 @@ spec:
   compositeTypeRef:
     apiVersion: aws.platformref.upbound.io/v1alpha1
     kind: XCluster
+  patchSets:
+  - name: specId
+    patches:
+    - fromFieldPath: spec.id
+      toFieldPath: spec.id
+  - name: region
+    patches:
+    - fromFieldPath: spec.parameters.region
+      toFieldPath: spec.parameters.region
   resources:
     - base:
         apiVersion: aws.platformref.upbound.io/v1alpha1
         kind: XNetwork
       patches:
-        - fromFieldPath: spec.id
-          toFieldPath: spec.id
+        - type: PatchSet
+          patchSetName: specId
```

```suggestion
        - fromFieldPath: spec.id
```
If you are trying to keep things DRY with this PatchSet then in this case, it is better to use one-liner as we can omit `toFieldPath` see https://docs.crossplane.io/v1.12/concepts/composition/#hl-7-110